app/src/test/java/com/onelist/ItemUnitTest.kt

```diff
@@ -1,15 +1,47 @@
 package com.onelist
 
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
 import com.onelist.dto.Item
-import junit.framework.TestCase.assertEquals
+import com.onelist.service.ItemService
+import junit.framework.TestCase.*
+import kotlinx.coroutines.test.runTest
+import org.junit.Rule
 import org.junit.Test
+import org.junit.rules.TestRule
 
 class ItemUnitTest {
 
```

Comment:
Followed the given-when-then schema for unit testing that uses real data and is wired compared to a generic hard-coded unit test